diesel/src/sqlite/mod.rs

```diff
@@ -12,5 +12,19 @@ pub mod query_builder;
 
 pub use self::backend::{Sqlite, SqliteType};
 pub use self::connection::SqliteConnection;
-pub use self::connection::raw::Aggregator;
 pub use self::query_builder::SqliteQueryBuilder;
+
+/// The implementation details for a SQLite aggregate function
+///
+/// This trait is to be used in conjunction with the `sql_function!`
+/// macro for defining a custom SQLite aggregate function.
+pub trait SqliteAggregateFunction<Args>: Default {
```

Comment:
Some small suggestions on the wording, but I'm not 100% happy with my formulations either.

```suggestion
/// Trait for the implementation of a SQLite aggregate function
///
/// This trait is to be used in conjunction with the `sql_function!`
/// macro for defining a custom SQLite aggregate function. See
/// the documentation [there](link to sql_function!) for details. 
pub trait SqliteAggregateFunction<Args>: Default {
```